isis/src/base/objs/BulletDskShape/BulletDskShape.cpp

```diff
@@ -155,14 +145,29 @@ namespace Isis {
 /**
  * @brief Load the contents of a NAIF DSK and create a Bullet triangle mesh
  *
+ * Do realtime validation. DSK limits triangles to 32M and vertices to ~16M vectors
+ * so ensure each segment shape can fit in a single Bullet part.  See
+ * https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/C/req/dsk.html#Appendix%20B%20---%20DSK%20Subsystem%20Limits
+ *
+ * Bullet limits in the repo are currently set at 4 parts and 134,217,728 triangles. However,
+ * that version has not been release since Aug 2022. Hence, we must ensure that all segment shapes
+ * fit into the bullet mesh mapping scheme which did have 1024 parts that supports no more
+ * than 2,097,152 triangles. At ant rate, loading DSKs must consider these limits when mapping to
+ * bullet use. So DSK support is limited to at most 1024 segments where each segment has no more
+ * than 2,097,152. Since 
```

Comment:
This comment likely needs a bit of work when it comes to: "ant rate", "2,097,152.", and "Since.", so fixing one typo, completing one sentence, and removing unneeded word at the end.